### PR TITLE
Fixed email attachment icon

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Fixed an issue with the email attachment icon not appearing in the email list-view.

## How To Test This

1. Set up inbound email.
2. Receive emails with attachments.
3. Check that all emails containing attachments show the icon in the list-view regardless of import status.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Mac-Rae** [comment]
Assessed :+1: 

Do not merge until #7554 is merged

**Comment by Dillon-Brown** [comment]
@Mac-Rae Other way around, this needs merged before #7554 :). I've pushed a change to make the list-view detection a bit stricter and updated the other PR to fix a typo.

File: modules/InboundEmail/InboundEmail.php [hunk 1]
```diff
@@ -548,22 +548,21 @@ function ($a, $b) use ($sortCRM) {  // defaults to DESC order
      * @param $imapStructure
      * @return bool
      */
-    private function mesageStructureHasAttachment($imapStructure)
+    public function mesageStructureHasAttachment($imapStructure)
```
Comment (pgorod) [inline modules/InboundEmail/InboundEmail.php]: This is just a detail but the typo of `mesage` instead of `message` might be worth correcting?

**Comment by pgorod** [comment]
@Mac-Rae @Dillon-Brown 	I think you guys both have it wrong.

This one _is_ #7554. So it is impossible to merge this one either before or after #7554.

You probably meant #7564 and had a Friday afternoon senior moment

**Pull request merged**